python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -475,8 +471,10 @@ class RandomForestRegressor(Base):
             input_to_dev_array(X, order='F')
 
         if self.dtype == np.float64:
-            warnings.warn("In order to run predict on the GPU convert"
-                          " the data to float32")
+            warnings.warn("Model is being trained with float64 data, which \
+                          means only CPU based predict can be used. \
+                          To use GPU based predict train use float32 training \
```

Comment:
Current text is a little hard to read. How about: `To use GPU-based prediction, first train using float 32 data to fit the estimator`